CHANGELOG.md

```diff
@@ -1,12 +1,71 @@
 # osquery Changelog
 
+<a name="4.5.1"></a>
+## [4.5.1](https://github.com/osquery/osquery/releases/tag/4.5.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.5.0...4.5.1)
+
+
+### New Features
+
+### Under the Hood improvements
+
+- carver: Improve tests by faking postCarve ([#6659](https://github.com/osquery/osquery/pull/6659))
+- carver: Emit an error if the carve function is disabled ([#6658](https://github.com/osquery/osquery/pull/6658))
+- carver: Update carves specs to allow full scan ([#6657](https://github.com/osquery/osquery/pull/6657))
+- carver: Update table to use JSON ([#6656](https://github.com/osquery/osquery/pull/6656))
+- Improve performance and accuracy of Windows registry querying ([#6647](https://github.com/osquery/osquery/pull/6647))
```

Comment:
The other line items all seem to have `<category>:` prefixes, perhaps this could use `performance:`?

Reply:
I think the prefixes are an anti pattern. I'll remove them all